CONTRIBUTING.md

```diff
@@ -19,10 +19,10 @@ Read this guide on how to do that.
 
 **Prerequisites**: familiarity with [GitHub Pull Requests](https://help.github.com/articles/using-pull-requests)
 
-If you want to contribute code, fork the repository and make a pull-request
-with your changes. A developer of the [open-build-service team](https://github.com/orgs/openSUSE/teams/open-build-service)
-will review your pull-request. And if the pull request gets a positive review
-the reviewer will merge it.
+If you want to contribute code, check the list of issues that we think are a
+[easy to start with](https://github.com/openSUSE/open-build-service/contribute).
+
+Then fork the repository and make a pull-request with your changes. A developer of the [open-build-service team](https://github.com/orgs/openSUSE/teams/open-build-service) will review your pull-request. And if the pull request gets a positive review the reviewer will merge it.
```

Comment:
```suggestion
Then fork the repository and make a pull request with your changes. A developer of the [open-build-service team](https://github.com/orgs/openSUSE/teams/open-build-service) will review your pull request. And if the pull request gets a positive review, the reviewer will merge it.
```

The link https://github.com/orgs/openSUSE/teams/open-build-service redirects to GitHub's login page for anonymous users.